ui/Badges/Badges.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+interface BadgesProps {
+  tags: string[];
+}
+
+const Badges: React.FC<BadgesProps> = ({ tags }) => {
```

Comment:
obsłuż sytuację w razie jakby tagów nie było, czli `({ tags = [] })`

Reply:
Jest to obsługiwane. W przypadku pustej listy nic nie zwróci. Jest nawet test do tego:
`
describe('Badges test', () => {
  it('empty list', () => {
    const { queryByTestId } = render(<Badges tags={[]} />);
    const tags = queryByTestId('tag');
    expect(tags).toBeNull();
  });
`